.travis.yml

```diff
@@ -10,14 +10,19 @@ php:
 # See: http://docs.travis-ci.com/user/workers/container-based-infrastructure/
 sudo: false
 
+## Cache composer
+cache:
+  directories:
+    - $HOME/.composer/cache
+
 matrix:
   include:
     - php: 5.5
       env: 'COMPOSER_FLAGS="--prefer-stable --prefer-lowest"'
 
 before_script:
   - travis_retry composer self-update
-  - travis_retry composer update ${COMPOSER_FLAGS} --no-interaction --prefer-source
```

Comment:
Yes see this thread; https://github.com/composer/composer/issues/4884#issuecomment-195229989

No official response yet (except from the support ticket response + employee responses..)

And you can test it by downloading without token + checking your limit; https://api.github.com/rate_limit